### PR TITLE
Add export only parameters for Backend and Frontend

### PR DESCRIPTION
**Pull request by r0b2g1t**
### All Submissions:

- [x] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [x] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

- [x] New feature (non-breaking change which adds functionality)

### New Feature Submission:

- [x] Does your submission pass the tests? 
- [x] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you successfully ran tests with your changes locally?

This PR adds the parameters to pc cli tool to export only the backend or frontend zip packages. Which was asked for in the issue #496.

### CODE REVIEW

File: pynecone/utils.py [hunk 1]
```diff
@@ -503,9 +507,40 @@ def export_app(app: App, zip: bool = False):
...
     if zip:
         if os.name == "posix":
-            cmd = r"cd .web/_static && zip -r ../../frontend.zip ./* && cd ../.. && zip -r backend.zip ./* -x .web/\* ./assets\* ./frontend.zip\* ./backend.zip\*"
+            posix_export(backend, frontend)
         if os.name == "nt":
-            cmd = r'''powershell -Command "Set-Location .web/_static; Compress-Archive -Path .\* -DestinationPath ..\..\frontend.zip -Force;cd ..\..;Get-ChildItem .\* -Directory  | where {`$_.Name -notin @('.web', 'assets', 'frontend.zip', 'backend.zip')} | Compress-Archive -DestinationPath backend.zip -Update"'''
+            nt_export(backend, frontend)
+
+
+def nt_export(backend, frontend):
```
Comment (picklelo) [inline pynecone/utils.py:515]: Can we add type annotations to the args here

File: pynecone/utils.py [hunk 2]
```diff
@@ -503,9 +507,40 @@ def export_app(app: App, zip: bool = False):
...
+    cmd = r""
+    if frontend:
+        cmd = r'''powershell -Command "Set-Location .web/_static; Compress-Archive -Path .\* -DestinationPath ..\..\frontend.zip -Force"'''
+        os.system(cmd)
+    if backend:
+        cmd = r'''powershell -Command "Get-ChildItem .\* -Directory  | where {`$_.Name -notin @('.web', 'assets', 'frontend.zip', 'backend.zip')} | Compress-Archive -DestinationPath backend.zip -Update"'''
+        os.system(cmd)
+
+
+def posix_export(backend, frontend):
```
Comment (picklelo) [inline pynecone/utils.py:531]: And here also

**Review by picklelo — COMMENTED** [review]
Nice! Just a minor comment

File: pynecone/utils.py (hunk 1 above)
Comment (r0b2g1t) [reply pynecone/utils.py]: done

File: pynecone/utils.py (hunk 2 above)
Comment (r0b2g1t) [reply pynecone/utils.py]: done too :-)

**Review by picklelo — APPROVED** [review]
Sweet!

**Pull request merged**